### PR TITLE
AmountInput component

### PR DESCRIPTION
**Pull request by srdjan-susa**
AmountInput component

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,19 +1,18 @@
...
 import BRD from './assets/icons/BRD.svg';
 import React from 'react';
-import { RegionCardData } from './components/RegionCard/RegionCard';
+import { RegionCardData } from './types/types';
 
 function App() {
-  // const [count, setCount] = useState(0)
   const selectOptions = [
-    { value: 'option1', label: 'Option 1',},
+    { value: 'option1', label: 'Option 1', icon: BRD },
```
Comment (Szegoo) [inline src/App.tsx:15]:
```suggestion
    { value: 'option1', label: 'Option 1', icon: KSM },
```

File: src/components/AmountInput/AmountInput.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+  useEffect(() => {
+    if (currencyOptions.length > 0) {
+      setCurrency(currencyOptions[0].value); // Automatically set the first option as selected
+      if (onCurrencyChange) onCurrencyChange(currencyOptions[0].value);
+    }
+  }, [currencyOptions, onCurrencyChange]);
+
+  const handleAmountChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const value = e.target.value.replace(/[^0-9]/g, ''); // Blocking e input
```
Comment (Szegoo) [inline src/components/AmountInput/AmountInput.tsx:37]: Can't enter decimal numbers

File: src/App.tsx (hunk 1 above)
Comment (srdjan-susa) [reply src/App.tsx]: Added changes with last commit

**Review by Szegoo — APPROVED** [review]
Looks good to me. Feel free to merge.

**Pull request merged**